patches/tModLoader/Terraria/ModLoader/Default/UnloadedChestInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Terraria.ModLoader.IO;
+using System.Collections.Generic;
+using Terraria.ID;
+
+namespace Terraria.ModLoader.Default
+{
+	internal class UnloadedChestInfo : UnloadedInfo 
+	{
+		internal byte ChestStyle; 
```

Comment:
This and many other fields should be in camelCase.

Reply:
Either do like Mirsario said and also add `readonly` to it.
`internal byte readonly chestStyle;`

Or just make it a readonly auto property and leave it UpperCamelCase (considering that it's non-private).
`internal byte ChestStyle { get; }`

Reply:
Fixed this line to be internal readonly byte chestStyle.